### ISSUE TITLE
_ Have a question? Search up there first ☝️ (remove all the filters)

### ISSUE THREAD

**Issue by steven-hh-ding** [CISC-873/Information#1]
Please search for closed issues for asked questions first !!!

**Comment by abigailkearney** [comment]
Where can we find the class survey? Thanks!

**Comment by taylorfountain** [comment]
Will lecture recordings be posted?

**Comment by theriaultr** [comment]
In Assignment 1 it states we need to try at least 3 preprocessing techniques. Would for example eliminating variables we shouldn't consider and replace NaN values be considered 2 different techniques?

**Comment by steven-hh-ding** [comment]
ah questions posted here! I was wondering.. hmm I got a notification but where it is. I thought questions are in new issues (so we can close them once answered)

@abigailkearney 
now available on the front-page.

@taylorfountain 
yah! V3 on the way

@theriaultr 
yes! even replacing NaN with different algorithms are considered different. but I encourage to try complicated one (so in the future when you do have to use a complicated one, you can quickly refer to your own example)

**Comment by theriaultr** [comment]
Just to clarify, after trying at least 3 pre-processing methods are we supposed to pick just 1 of them to use moving forward, or do we need to use each one?
Thank you for your help!

**Comment by steven-hh-ding** [comment]
@theriaultr  yup just letting you to be familiarize (or have a place to refer to as example) how to do different preprocessing quickly, if needed. preprocessing is also part of the configuration/hyperparameter of your solution that can be tuned.
